crates/symbolicator/src/services/download/mod.rs

```diff
@@ -80,14 +81,28 @@ impl DownloadService {
         let trusted_client = crate::utils::http::create_client(&config, true);
         let restricted_client = crate::utils::http::create_client(&config, false);
 
+        let Config {
+            download_timeout,
+            streaming_timeout,
+            ..
+        } = *config;
+
         Arc::new(Self {
             config,
             worker: tokio::runtime::Handle::current(),
-            sentry: sentry::SentryDownloader::new(trusted_client),
-            http: http::HttpDownloader::new(restricted_client.clone()),
-            s3: s3::S3Downloader::new(),
-            gcs: gcs::GcsDownloader::new(restricted_client),
-            fs: filesystem::FilesystemDownloader::new(),
+            sentry: sentry::SentryDownloader::new(
+                trusted_client,
+                download_timeout,
```

Comment:
I would just pass a `&Config` to these constructors. They can copy out anything they want from there.

Reply:
heh, this is the opposite of something i recently said.  i prefer the download service (or any other service, or more generically a piece of code) not to have access to the global config directly because it is a prime way of starting to leak abstractions and accidentally introducing tight coupling.